src/main/io/serial_cli.c

```diff
@@ -970,6 +974,82 @@ static void cliSerial(char *cmdline)
 
 }
 
+static void cliSerialPassthrough(char *cmdline)
+{
+    if (isEmpty(cmdline)) {
+        cliShowParseError();
+        return;
+    }
+
+    int id = -1;
+    uint32_t baud = 0;
+    baudRate_e baudIdx = BAUD_AUTO;
+    unsigned mode = 0;
+    char* tok = strtok(cmdline, " ");
+    int index = 0;
+
+    while (tok != NULL) {
+        switch(index) {
+            case 0:
+                id = atoi(tok);
+                break;
+            case 1:
+                baud = atoi(tok);
+                baudIdx = lookupBaudRateIndex(baud);
```

Comment:
Is it really necessary to force one of enumerated baudrates? Let user shoot his leg if he insists ...